source/daemon/src/daemon.cpp

```diff
@@ -2494,28 +2247,6 @@ void Daemon::onLogParameterPolled(){
         qDebug() << " file size: " << fileHandler->dataFileInfo().size()/(1024*1024) << "MiB";
     }
 
-//      Since Linux 2.3.23 (i386) and Linux 2.3.48 (all architectures) the
```

Comment:
This comment block was intentionally copied from the kernel docs here in order to see the structure and the other available fields. Imho it serves some useful purpose, so i would ask to leave it.

Reply:
Fixed with https://github.com/MuonPi/muondetector/commit/246663e4ff2e27037d5c34e004d0f7785ca42e01#diff-41cf6794ba4200b839c53531555f0f3998df4cbb01a4d5cb0b94e3ca5e23947d